invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/components/ResultsList/ResultsList.js

```diff
@@ -15,33 +15,84 @@ export class ResultsList extends Component {
     this.fetchPatronCurrentLoans = this.props.fetchPatronCurrentLoans;
   }
 
+  state = { open: false };
+
+  openModal = () => this.setState({ open: true });
+
+  closeModal = () => this.setState({ open: false });
```

Comment:
Usually for boolean states what I do instead of explicitly define open and close I toggle.

`toggleModal = () => this.setState({ open: !this.state.open })` and then you can reuse `toggleModal` to replace each of the `openModal` and `closeModal`

on another note instead of calling it `open` I would prefer something more self descriptive like `isModalOpen`.